public/index.html

```diff
@@ -3,7 +3,7 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Comment Section Module</title>
```

Comment:
Yay you changed this! I'm running the older commit locally and this was going to be my first suggestion 👍